### PR TITLE
Cms/383,384,388,389

### PR DESCRIPTION
**Pull request by Legai**
I branched off from Vlad's branch: CMS/334, created pug-files, controllers, added css and images file to appropriate directories.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+*.idea*
+.idea/
```
Comment (102) [inline .gitignore]: `.idea` is enough

File: backend/app.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+app.set('views', './app/views');
+app.set('view engine', 'pug');
+
+app.use(express.static('public'));
+
+app.use(router);
+
+app.listen(3005, () => {
+    console.log('Server is started!!!');
+});
```
Comment (102) [inline backend/app.js]: tl;dr add newline to the end of files

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: backend/app/views/pages/advantages.pug [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                            .col-left-block
+                                .adv-circle(style="background-image: url('/images/3.png');")
+                            .col-right-block
+                                .col-name Individual Approach
+                                .col-text
+                                    | Our company works according to the principle of
+                                    | individual approach to every client.
+                                    | This method lets us to get success in problems of all levels.
+                                    
+
```
Comment (102) [inline backend/app/views/pages/advantages.pug]: You should try to avoid extra empty lines & trailing spaces

**Comment by maxmalov** [comment]
Tickets: 383,384,388,389
Files changed: 42
Diff: +1,013 −0

Total stats are telling me this PR will never pass code review, since lots of stuff are combined here. As I mentioned before PR should be as less as possible, because the less changes you're going to ship the less comments you'll take. Ideally each PR should be atomic and solve a single issue. In this case there are 4 of them, so I'm expecting at least 4 PRs.

You should review the whole change set and try to decompose it to a different change sets. Since, you depend on Vlad's code you should be interested in this PR too. This is where you 2 should cooperate with each other and ship the common code faster, so it won't block anybody further.

**Pull request closed without merging**